src/Sentry/Sentry.csproj

```diff
@@ -70,9 +77,9 @@
     <Reference Include="System.Net.Http" />
   </ItemGroup>
 
-  <!-- System.Text.Json is already included with .NET 5 and higher. Add a direct reference for other targets. -->
-  <ItemGroup Condition="!$(TargetFramework.StartsWith('net5')) and !$(TargetFramework.StartsWith('net6'))">
-    <PackageReference Include="System.Text.Json" Version="5.0.2" />
+  <!-- Ensure at least version 6 of System.Text.Json so we have JsonSerializationContext available -->
+  <ItemGroup Condition="!$(TargetFramework.StartsWith('net6')) and !$(TargetFramework.StartsWith('net7')) and !$(TargetFramework.StartsWith('net8'))">
+    <PackageReference Include="System.Text.Json" Version="6.0.8" />
```

Comment:
How come the original dependency is now not necessary for .net framework **lower than** net5? That's whet the original piece of the code was fore, right? I understand the new code, just don't get why we could remove the previous one. I'd have expected we need both.